### PR TITLE
Convert all resources to JSON

### PR DESCRIPTION
**Pull request by noemide**
Converted remaining resources to JSON & Fixed datatype slicing errors caused by previous version of Forge.

### CODE REVIEW

File: profile/GbaCondition.StructureDefinition.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        "max": "0"
+      },
+      {
+        "id": "Condition.subject",
+        "path": "Condition.subject",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "http://hl7.org/fhir/StructureDefinition/Patient"
```
Comment (alexanderkiel) [inline profile/GbaCondition.StructureDefinition.json]: That could point to our patient profile - right?

File: profile/GbaObservationSmoker.StructureDefinition.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        "max": "0"
+      },
+      {
+        "id": "Observation.subject",
+        "path": "Observation.subject",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "http://hl7.org/fhir/StructureDefinition/Patient"
```
Comment (alexanderkiel) [inline profile/GbaObservationSmoker.StructureDefinition.json]: Here also out patient profile.

File: profile/GbaPatient.StructureDefinition.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      },
+      {
+        "id": "Patient.telecom",
+        "path": "Patient.telecom",
+        "max": "0"
+      },
+      {
+        "id": "Patient.gender",
+        "path": "Patient.gender",
+        "short": "Male | Female | Other/intersexual | Not given"
```
Comment (alexanderkiel) [inline profile/GbaPatient.StructureDefinition.json]: We should not have a custom short description here. May remove the whole data element?

File: profile/GbaSpecimen.StructureDefinition.json [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+          {
+            "code": "Tissue Paxgene or else",
+            "display": "Tissue Paxgene or else"
+          },
+          {
+            "code": "Other tissue storage",
+            "display": "Other tissue storage"
+          }
+        ]
+      },
```
Comment (alexanderkiel) [inline profile/GbaSpecimen.StructureDefinition.json]: That looks wrong. We have already the ValueSet binding.

File: profile/GbaSpecimen.StructureDefinition.json [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+        "max": "0"
+      },
+      {
+        "id": "Specimen.subject",
+        "path": "Specimen.subject",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "http://hl7.org/fhir/StructureDefinition/Patient"
```
Comment (alexanderkiel) [inline profile/GbaSpecimen.StructureDefinition.json]: Our patient.

File: profile/GbaSpecimen.StructureDefinition.json [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+          {
+            "code": "Liquid nitrogen",
+            "display": "Liquid nitrogen"
+          },
+          {
+            "code": "Other storage temperature",
+            "display": "Other storage temperature"
+          }
+        ]
+      },
```
Comment (alexanderkiel) [inline profile/GbaSpecimen.StructureDefinition.json]: Looks wrong.

File: profile/GbaSpecimen.StructureDefinition.json [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+        "path": "Specimen.collection.bodySite.coding.userSelected",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.collection.bodySite.text",
+        "path": "Specimen.collection.bodySite.text",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.collection.fastingStatus[x]:fastingStatusCodeableConcept",
```
Comment (alexanderkiel) [inline profile/GbaSpecimen.StructureDefinition.json]: Slicing entry without main slice definition?

File: profile/GbaSpecimen.StructureDefinition.json (hunk 4 above)
Comment (noemide) [reply profile/GbaSpecimen.StructureDefinition.json]: Right! Removed it.

File: profile/GbaSpecimen.StructureDefinition.json (hunk 6 above)
Comment (noemide) [reply profile/GbaSpecimen.StructureDefinition.json]: Also removed.

File: profile/GbaPatient.StructureDefinition.json (hunk 3 above)
Comment (noemide) [reply profile/GbaPatient.StructureDefinition.json]: Agreed.

**Comment by alexanderkiel** [comment]
Very good thanks!

**Pull request merged**